src/components/Error/ErrorBoundary.tsx

```diff
@@ -11,6 +12,14 @@ export interface ErrorBoundaryProps {
 export const ErrorBoundary = (props: PropsWithChildren<ErrorBoundaryProps>) => {
   const { children, headline, subhead, body } = props;
 
+  if (sentryDsn) {
+    Sentry.init({
+      dsn: sentryDsn,
+      environment: vercelEnv,
+      release: commitSha
+    });
+  }
```

Comment:
This needs to be initialized in the entry point for the react (https://docs.sentry.io/platforms/javascript/guides/react/#configure) so that Sentry can capture all the relevant metadata.

Reply:
Next.js complicates things a bit. You'll see where we handle the entry points. The official plugin does webpack injection, which I'd rather not replicate for the sake of simplicity.

Reply:
The init above was moved to `_app.ts` and is working well, fyi. Still need an inline init in the node wrapper.